e2e/workspace-create/global-setup.ts

```diff
@@ -0,0 +1,48 @@
+import { join } from 'path';
+import * as detectPort from 'detect-port';
+import { ChildProcess, execSync, spawn } from 'child_process';
+
+export default async function () {
+  const port = await detectPort(4872);
+  const registry = `http://localhost:${port}`;
+  process.env.npm_config_registry = registry;
```

Comment:
Can we add a console.log here which says that npm + yarn have been configured to use the local registry?

Reply:
chnaged.